aws/resource_aws_resourcegroups_group.go

```diff
@@ -176,3 +216,45 @@ func resourceAwsResourceGroupsGroupDelete(d *schema.ResourceData, meta interface
 
 	return nil
 }
+
+func tagsToMapRG(ts map[string]interface{}) map[string]string {
```

Comment:
I think you can reuse the functionality in [`tagsGeneric.go`](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/tagsGeneric.go) here.

Reply:
Many thanks !!!! Good to know since I'm struggle with the Tag function !!